### DISCUSSION TITLE
Shimmer Rewards gone

### DISCUSSION THREAD

**Discussion by tn2345** [iotaledger/firefly#7590] [Help & Support]
Hello everyone,
after downloading the new version of firefly iota (2.0.0) my shimmer rewards aren't visible anymore. I have downloaded firefly shimmer and tried to find my shimmer rewards with the ledger but there are different errors like: 

Ledger transport error or Account 2 not found or Account 1 not found or Ledger not found

can you please help me?

**Comment by begonaalvarezd** [comment]
Hi, we found a bug in the shimmer claiming flow and we are preparing a release this week to fix it, the fix is already merged so we are just preparing the testing and release, firefly shimmmer V2.1.9 will have the fix

Will close the discussion because it has been fixed by https://github.com/iotaledger/firefly/pull/7568
